plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -926,10 +929,12 @@ static void
 fu_uefi_capsule_plugin_test_secure_boot(FuPlugin *plugin)
 {
 	FuEfivars *efivars = fu_context_get_efivars(fu_plugin_get_context(plugin));
-	const gchar *result_str = "Disabled";
-	if (fu_efivars_secure_boot_enabled(efivars, NULL))
-		result_str = "Enabled";
-	fu_plugin_add_report_metadata(plugin, "SecureBoot", result_str);
+	gboolean secureboot_enabled = FALSE;
+
+	fu_efivars_get_secure_boot(efivars, &secureboot_enabled, NULL);
```

Comment:
I don't think it's worth ignoring the error here, at least log it

Reply:
maybe set the report attr to `Unknown`? That'd be an interesting datapoint.

Reply:
Yeah; that's what I was thinking - we should know if there is such an error here ever.

Reply:
I put the error message there -- that way we can tell the difference between an empty var and a missing var.